Libraries/Components/View/ViewAccessibility.js

```diff
@@ -12,11 +12,6 @@
 
 import type {SyntheticEvent} from '../../Types/CoreEventTypes';
 
-// [TODO(android ISS)
-export type AccessibilityNodeInfoProp = {
-  clickable: boolean,
-}; // ]TODO(android ISS)
```

Comment:
@mganandraj / @rasaha91 This is an Office Android change not in a patch that I am removing

Reply:
Removing in #1591 instead. Will wait to merge this PR to avoid merge conflicts.